server/container_create.go

```diff
@@ -344,6 +344,7 @@ func addDevices(sb *sandbox.Sandbox, containerConfig *pb.ContainerConfig, specge
 
 				// mount the internal devices recursively
 				filepath.Walk(path, func(dpath string, f os.FileInfo, e error) error {
+					_ = e // we're not checking this error, but are immediately shadowing it
```

Comment:
Should we at least log these?

Reply:
this was found via linting, and i'm hesistant to return if error, but might could log to Info?